mmu-to-hats/README.md

```diff
@@ -42,6 +43,9 @@ Note that we need to return a `pyarrow.Table` with exactly the same output as th
  - verification/download_sdss.sh
  - process_sdss_using_datasets.py
 
+Please note, that most of the classes are vibe-coded, and not verified yet.
+The only verified transformation class is SDSS.
```

Comment:
Maybe, in a later PR we can create a list of properly reviewed and verified catalog transformers!